### PR TITLE
Fix a typo in dtksettings: powerfull -> powerful

### PR DESCRIPTION
**Pull request by felixonmars**

### CODE REVIEW

File: rpms/deepin_project/dtksettings.spec [hunk 1]
```diff
@@ -4,14 +4,14 @@
 Name:           dtksettings
 Version:        0.1.6
 Release:        1.git%{_shortcommit}%{?dist}
-Summary:        DtkSettings is a powerfull tool to generation config form json
+Summary:        DtkSettings is a powerful tool to generation config form json
```
Comment (alick) [inline rpms/deepin_project/dtksettings.spec]: More fixes: to *generate* configuration *from* JSON

File: rpms/deepin_project/dtksettings.spec [hunk 2]
```diff
@@ -4,14 +4,14 @@
...
 License:        GPLv3
 URL:            https://github.com/linuxdeepin/dtksettings
 Source0:        %{url}/archive/%{_commit}/%{name}-%{_shortcommit}.tar.gz
 BuildRequires:  qt5-qtbase-devel
 
 %description
-DtkSettings is a powerfull tool to generation config form json.
+DtkSettings is a powerful tool to generation config form json.
```
Comment (alick) [inline rpms/deepin_project/dtksettings.spec]: Ditto.

**Comment by alick** [comment]
Commented in diff.

**Comment by felixonmars** [comment]
@alick Both updated :)

**Pull request merged**